lib/libzfs/libzfs_pool.c

```diff
@@ -4097,9 +4097,33 @@ zbookmark_mem_compare(const void *a, const void *b)
 	return (memcmp(a, b, sizeof (zbookmark_phys_t)));
 }
 
+/*
+ * Either adds both the arrays in the nvlist or none.
+ */
+static void
+add_both_arrays_or_none(nvlist_t *object_nv, uint64_t *object_block_ids,
```

Comment:
Would be better to rename this to be more specific, something like - `add_error_block_ids_and_levels`

Reply:
Done